core/fuzz/fuzz_reader.rs

```diff
@@ -172,10 +178,15 @@ async fn fuzz_reader_process(input: FuzzInput, op: &Operator, name: &str) -> Res
         .await
         .unwrap_or_else(|_| panic!("{} write must succeed", name));
 
-    let mut o = op
-        .range_reader(&path, 0..len as u64)
-        .await
-        .unwrap_or_else(|_| panic!("{} init range_reader must succeed", name));
+    let mut o = if is_range {
+        op.range_reader(&path, 0..len as u64)
```

Comment:
`op.range_reader(&path, 0..len as u64)` is exactly the same with `op.reader(&path)`. We don't need to test in this way. Instead, we should generate a random range for `range_reader` to test.

`fuzz_range_reader` should be a new fuzz apart from `fuzz_reader`.

Reply:
solved by https://github.com/apache/incubator-opendal/pull/2706/commits/fad270eeb06cbe536f564285a5ca298feb52f618